packages/echo/src/express.ts

```diff
@@ -8,9 +9,11 @@ import { type SupportedFrameworkName } from './types';
 export const frameworkName: SupportedFrameworkName = 'express';
 
 export const serve = (options: ServeHandlerOptions): any => {
+  const client = options.client ? options.client : new Echo();
```

Comment:
I think we can handle this instantiation inside the `EchoRequestHandler`, the `...options` spread will ensure the client is injected if available, and we won't need to modify any framework files.

Reply:
Yep, much better approach